tests/components/geofency/test_init.py

```diff
@@ -113,34 +113,34 @@ def mock_dev_track(mock_device_tracker_conf):
 
 
 @pytest.fixture
-def geofency_client(loop, hass, aiohttp_client):
+async def geofency_client(loop, hass, aiohttp_client):
```

Comment:
I thought fixtures weren't allowed to be async?

Reply:
This feature provided by `pytest-asyncio` plugin